test/parallel/test-http2-capture-rejection.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+const common = require('../common');
```

Comment:
Insert `hasCrypto` check here per the linter.

```suggestion
const common = require('../common');
if (!common.hasCrypto)
  common.skip('missing crypto');
```